player-tester/src/chart.js

```diff
@@ -349,3 +348,19 @@ export function stopUpdatingChart() {
   clearTimeout(chartUpdatingIntervalId);
   chartIsUpdating = false;
 }
+
+
+export function exportChart(playerName) {
+  const data = {player: "rxplayer", data: config}
+  fetch(CDN_SERVER_URL+"/report", {
```

Comment:
Why go through the back-end?
Why not just asking to store directly the JSON here? So the file is directly in the right directory without user input?